_faqs-test-based/05_trial.md

```diff
@@ -0,0 +1,5 @@
+---
+# Question title. Plaintext only.
+question: "What’s the deal with the 30-day free trial?"
+---
+Each Snyk organisation can try out our paid-for plans for 30 days for free, without providing payment information. You can upgrade or pay during the trial to keep monitoring your private projects beyond the trial, otherwise we will downgrade you to the free plan when the trial ends.
```

Comment:
Suggestion:

You can upgrade or pay during the trial to keep ~monitoring~ **testing** your private projects beyond the trial, otherwise we will downgrade you to the free plan when the trial ends.